manifests/baremetal/keepalived.yaml

```diff
@@ -86,7 +97,7 @@ spec:
         - /bin/bash
         - -c
         - |
-          [[ -s /etc/keepalived/keepalived.conf ]] || \
+          [[ -s /etc/keepalived/stopped ]] || \
```

Comment:
Should this be an added check instead of replacing the keepalived.conf check? We still don't want to start liveness checks until after the config has been rendered.

Reply:
Yep, maybe it's better to keep also this check.
Will update
10x